python/paddle/distributed/fleet/meta_optimizers/lars_optimizer.py

```diff
@@ -68,6 +71,9 @@ def backward(self,
         return self.lars_opt.backward(loss, startup_program, parameter_list,
                                       no_grad_set, callbacks)
 
+    def apply_gradients(self, params_grads):
```

Comment:
comments required to explain why you use apply_gradient here.

Reply:
explained in the code